lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -50,7 +62,7 @@
   %endif
 
     <div class="request-response msg msg-confirm copy" id="report-request-response"></div>
-    <div class="request-response-error msg msg-warning copy" id="report-request-response-error"></div>
+    <div class="request-response-error msg msg-error copy" id="report-request-response-error"></div>
```

Comment:
What does the error message look like with `msg-error` styling?

Is there anything that would be trying to populate the `msg-warning` div that now would not be able to?